docker-java/src/main/java/com/github/dockerjava/api/model/LogConfig.java

```diff
@@ -22,28 +29,28 @@
     private static final long serialVersionUID = 1L;
 
     @JsonProperty("Type")
-    public LoggingType type = null;
+    public Type type = null;
```

Comment:
this is a binary breaking change, could you please find a way to avoid it?

Reply:
Probably it should be a String at all, enum was made for known lists when docker didn't support custom types. And now here any String name could be.

Reply:
Probably LogConfig needs custom deserealiser with new string method and deprecate enum. Or at all convert field to string, but keep method that will either throw exception either return enum.